test.html

```diff
@@ -1,8 +1,8 @@
 <html>
   <head>
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/css/select2.min.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.5/css/select2.min.css" rel="stylesheet" />
```

Comment:
I tested that this change works in both 4.0.0 and 4.0.5 (the newest stable release).